examples/custom-data-types/src/index.ts

```diff
@@ -0,0 +1,51 @@
+import type {VaultFunction} from "@piiano/vault-bundles-types";
+
```

Comment:
Add comment about older Vault that did not accept these SSNs

Reply:
@ori-shalom something like: "Vaults prior to 1.10.4 had a stricter SSN validation. In this example we provided a new data type that has relaxed SSN validation that can work with earlier vaults."